src/poetry/console/commands/init.py

```diff
@@ -58,6 +58,7 @@ class InitCommand(Command):
             flag=False,
             multiple=True,
         ),
+        option("disable-package-mode", "-N", "Disables package mode.", flag=True),
```

Comment:
I am wondering about the short name. Does it stand for "non-package-mode"? If that is the case, should we name the option `non-package-mode`?